src/axom/inlet/docs/sphinx/mfem_coefficient_nested_documentation.rst

```diff
@@ -0,0 +1,83 @@
+.. |uncheck|    unicode:: U+2610 .. UNCHECKED BOX
+.. |check|      unicode:: U+2611 .. CHECKED BOX
+
+=====================================
+MFEM Coefficient Output (nested mode)
+=====================================
+.. list-table::
+   :widths: 25 50
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Name
+     - Description
+   * - `bcs`_
+     - List of boundary conditions
+
+
+
+---
+bcs
+---
+
+
+
+
+--------------------
+Collection contents:
+--------------------
+
+List of boundary conditions
+
+
+
+List of boundary conditions
```

Comment:
This is doubled for some reason.

Reply:
Good catch, ended up just being a copy-paste error!

Reply:
Duplication is still here.